superset/config.py

```diff
@@ -1360,7 +1360,7 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
 WEBDRIVER_AUTH_FUNC = None
 
 # Any config options to be passed as-is to the webdriver
-WEBDRIVER_CONFIGURATION: dict[Any, Any] = {"service_log_path": "/dev/null"}
+WEBDRIVER_CONFIGURATION: dict[Any, Any] = {}
```

Comment:
@gnought would you mind providing context for this change? Note irrespective of what the default values are this would still be deemed a breaking change given that the customizable Superset configurations are deemed a public interface.

CC @michael-s-molina for context regarding our recent discussion on breaking changes.

Reply:
@john-bodley 

In 4.9.0, `service_log_path` is one of parameters in `WebDriver` constructor.
https://github.com/SeleniumHQ/selenium/blob/bc7e0c7341fdde78cdd80ce572002a7866fd1769/py/selenium/webdriver/firefox/webdriver.py#L90-L110

However it is no longer in 4.10.0+.
https://github.com/SeleniumHQ/selenium/blob/c14d9678990942b93cb421c5567d0da7fb29c7bd/py/selenium/webdriver/firefox/webdriver.py#L50-L53

The following commit will support 4.10.0. It also opens for users to refine Webdrver `Options` and `Service`.
For sure, this will be a breaking change if users defining `WEBDRIVER_CONFIGURATION` themselves using 4.9.0 WebDriver defintion.